### ISSUE TITLE
Flag when a unit diverges from its scheduled route

### ISSUE THREAD

**Issue by api-h** [ucl-team-8/service#27]
Peter mentioned a core functionality of letting the user know when a car has diverged from it's planned route. Not sure if this is already done?

**Comment by danielgavrilov** [comment]
Do you mean checking if the trust reports match what's on the schedule? Did he really request that?

I don't see this mentioned in the requirements document and I don't remember him mentioning anything like this.

**Comment by api-h** [comment]
I think I remember him saying, it would be really nice it there was something that says, 'This train is going off schedule' or something along those lines, I could just be imagining things though but I don't think so

**Comment by danielgavrilov** [comment]
I missed one of the meetings, but I think he meant "tell us if it's not running the service it was supposed to run"? We didn't even get the schedule for Northern Rail, I extracted it from the API. 

(And I hope we both mean the same thing when we say **schedule**. I mean [this](https://github.com/ucl-team-8/service/tree/master/data/east_coast/diagrams))

**Comment by AxelGoetz** [comment]
I agree with @danielgavrilov, I can only remember him saying that we need to tell when a rolling stock is not running the service it was supposed to run.

**Comment by danielgavrilov** [comment]
I'll leave this as a _collaboration opportunity_.
